labs/multiverse/src/widgets/room_view/mod.rs

```diff
@@ -320,21 +319,14 @@ impl RoomView {
 
     /// Mark the currently selected room as read.
     pub async fn mark_as_read(&mut self) {
-        let selected = self.selected_room.as_deref();
-
-        let Some(room) = selected.and_then(|room_id| self.ui_rooms.lock().get(room_id).cloned())
-        else {
-            self.status_handle.set_message("missing room or nothing to show".to_owned());
-            return;
-        };
-
-        // Mark as read!
-        let Ok(timeline) = TimelineBuilder::new(&room).build().await else {
-            self.status_handle.set_message("failed creating timeline".to_owned());
+        let Some(sdk_timeline) = self.selected_room.as_deref().and_then(|room_id| {
```

Comment:
nit: let's call this `timeline`, not `sdk_timeline`; we're not in the FFI layer here.